features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/TimeBank.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder
+
+import com.datadog.tools.annotation.NoOpImplementation
+
+@NoOpImplementation
+internal interface TimeBank {
+
+    fun consume(executionTime: Long)
+
+    fun updateAndCheck(timestamp: Long): Boolean
```

Comment:
Maybe some description of that is needed. It is not clear what this class is supposed to do and what methods are doing by looking on the names.

Reply:
updated, please check it.